vm.go

```diff
@@ -327,17 +327,17 @@ func (vm *vm) setFuncName(s unistring.String) {
 
 func intToValue(i int64) Value {
 	if i >= -maxInt && i <= maxInt {
-		if i >= -128 && i <= 127 {
-			return intCache[i+128]
+		if i >= -8192 && i <= 8191 {
```

Comment:
[q/opt] Do you expect negative values to be used as much as positive ones? I was leaning towards caching the the values between -128 and somewhere between 10k-20k.